Makefile

```diff
@@ -866,6 +866,21 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint-clean:
+	$(RM) -r tools/remark-cli/node_modules
+	$(RM) -r tools/remark-preset-lint-node/node_modules
+
+mdlint-build:
+	if [ ! -d tools/remark-cli/node_modules ]; then \
+		cd tools/remark-cli && ../../$(NODE) ../../$(NPM) install; fi
```

Comment:
So I'm using `make` like so:
`d:\bin\MinGW\msys\1.0\bin\make PYTHON=python NODE=d:/bin/dev/node/node mdlint`
So `../../$(NODE)` is not working.
Maybe do something like
(non tested snippet)
```Makefile
mdlint-build:
	if [ ! -d tools/remark-cli/node_modules ]; then \
                ABSNODE := $(shell realpath $(NODE))
                cd tools/remark-cli && $(ABSNODE) ../../$(NPM) install; fi
```